src/core/p5.Element.js

```diff
@@ -122,7 +122,9 @@ p5.Element.prototype.parent = function(p) {
 /**
  *
  * Sets the ID of the element. If no ID argument is passed in, it instead
- * returns the current ID of the element.
+ * returns the current ID of the element.Note that only one element can have
+ * a particular id in a page. Refer to <a href="#/p5.Element/class">.class()</a>
+ * function to set names for multiple elements.
```

Comment:
I personally wouldn't call it "name" as for input fields "name" is a very specific attribute and may cause confusion here.

Reply:
@limzykenneth Thank you for the suggestion !
Would rephrasing the sentence as ,
".class() function can be used to identify multiple elements with the same class name." make it better ?

Reply:
Yeah that would be better